### ISSUE TITLE
Column resizing doesn't work with virtual scroll

### ISSUE THREAD

**Issue by TKul6** [primefaces/primeng#9622]
**I'm submitting a ...**  (check one with "x")
```
[X ] bug report => Search github for a similar issue or PR before submitting
[ ] feature request => Please check if request is not on the roadmap already https://github.com/primefaces/primeng/wiki/Roadmap
[ ] support request => Please do not submit support request here, instead see http://forum.primefaces.org/viewforum.php?f=35
```

**Plunkr Case (Bug Reports)**
https://primeng-tablecolresize-demo-nd18uv.stackblitz.io

**Current behavior**
When using the virtual scroll feature, there is an issue when attempting to resize the column. only the header changes the size but all the table body stay the same.

**Expected behavior**
When resizing the column, the entire column (including the data resizes).

**Minimal reproduction of the problem with instructions**
- Create table.
- Change the table to support virtual scrolling.
- Add `colgroup` template.
- Mark the `<th>` as resizable.

**What is the motivation / use case for changing the behavior?**
I'm using the VS feature, but I want also to let the user resize the columns.

**Please tell us about your environment:**

* **Angular version:** 10.2 

* **PrimeNG version:** 11.0.0-rc2

* **Browser:** [all]

 
* **Language:** [TypeScript 3.8]

* **Node (for AoT issues):** `node --version` =   12.18

Thanks you

**Comment by SPKorhonen** [comment]
I also encountered this problem. I think that issue #9598 also reports the same defect. The problem is fixed by PR #9601 which is a single line fix to a clear copy-paste mistake.

**Comment by yigitfindikli** [comment]
Duplicate of https://github.com/primefaces/primeng/issues/9598
